package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "silverstripe-userforms",
+  "version": "5.0.0",
+  "description": "UserForms module provides a visual form builder for the SilverStripe CMS. No coding required to build forms such as contact pages.",
+  "scripts": {
+    "build": "NODE_ENV=production webpack -p --bail --progress",
+    "watch": "NODE_ENV=development webpack --watch --progress",
```

Comment:
nowadays I prefix the `build` script with `yarn && NODE_ENV=p...` so that people do not forget to keep their `node_modules` up to date, it's quite common to add a new package in one pull request and forget about updating.

optionally include that for the `watch` script as well, and I'd recommend it.